pageserver/src/object_store.rs

```diff
@@ -60,4 +60,17 @@ pub trait ObjectStore: Send + Sync {
         dbnode: u32,
         lsn: Lsn,
     ) -> Result<HashSet<RelTag>>;
+
+    /// Iterate through all objects
+    ///
+    /// This is used to implement GC and preparing tarball for new node startup
```

Comment:
The callers rely on the order of the returned items, so better document that in the comment here.

Reply:
Sorry, will do